transformers/integration_tests/median_lift_test.go

```diff
@@ -0,0 +1,87 @@
+package integration_tests
+
+import (
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lib/pq"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/median_lift"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MedianLift EventTransformer", func() {
+	medianLiftConfig := event.TransformerConfig{
+		TransformerName:   constants.MedianLiftTable,
+		ContractAddresses: test_data.MedianAddresses(),
+		ContractAbi:       constants.MedianABI(),
+		Topic:             constants.MedianLiftSignature(),
+	}
+
+	It("transforms Median Lift log events", func() {
+		blockNumber := int64(8936530)
+		medianLiftConfig.StartingBlockNumber = blockNumber
+		medianLiftConfig.EndingBlockNumber = blockNumber
+
+		test_config.CleanTestDB(db)
+
+		header, headerErr := persistHeader(db, blockNumber, blockChain)
+		Expect(headerErr).NotTo(HaveOccurred())
+
+		// TODO: fetch event from blockchain once one exists
```

Comment:
I'd be a fan of marking this test as pending if we don't have an actual event to lookup from the chain, otherwise this is basically just reproducing the transformer test, right?

Reply:
I think the main difference is that the transformer test doesn't interact with the DB at all-- it still passes even if there's no migration adding these events' table. Personally I'm indifferent whether we mark the whole test pending or have a 'partial' test like this one, just thought it was worth pointing out.

Reply:
ah good point 👍 

for that matter this probably means we don't have tests that verify inserts work for all the integration tests that are currently pending 😱

Reply:
ha ha.. yeah, I think I'm going to make this pending since we may forget to update it when its not pending.